### ISSUE TITLE
Manifest version 2 is deprecated

### ISSUE THREAD

**Issue by geekwolverine** [CodingGarden/troll-garden#39]
Manifest version 2 is deprecated, and support will be removed in 2023. See https://developer.chrome.com/blog/mv2-transition/ for more details.

**Comment by geekwolverine** [comment]
![Screenshot_20220402_121201](https://user-images.githubusercontent.com/4777626/161370634-8277830d-a931-4c56-840a-bc1696d0c2f3.png)

**Comment by paoloricciuti** [comment]
Yeah @w3cj mentioned in the stream that he knows this but given the nature of this project he doesn't totally care about this. Also V3 remove support for background pages (which we use here) in favor of workers.

We could however think to rewrite this to be V3 compliant and upgrade to V3

**Comment by paoloricciuti** [comment]
On that note this could be an useful resource to migrate

https://developer.chrome.com/docs/extensions/mv3/migrating_to_service_workers/

P.s. another problem there was with V3 was the fact that the service worker will be shut down if not needed and CJ needed the background script to always run for the notifications. However reading this guide I discovered that you can replace setTimeout with the alarm api that does basically the same thing.
